examples/config_files/file_interface.toml

```diff
@@ -0,0 +1,32 @@
+# Example toml file as input for metasyn
+
+config_version = "1.2"
+
+[file]
+name = "csv"  # The file is a CSV file
```

Comment:
This sounds like filepath (file name). How about "type" instead?

Reply:
What about simply "file_format"?

Reply:
Done